meta/documents/changelog_de.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes für Ceres
 
+## v2.1.1 (2018-01-09) <a href="https://github.com/plentymarkets/plugin-ceres/compare/2.1.0...2.1.1" target="_blank"><b>Übersicht aller Änderungen</b></a>
+
+### Behoben
+
+- Es wurde ein Fehler behoben, bei dem Artikel ohne definierten URL-Pfad im **Warenkorb** nicht im Warenkorb angezeigt wurden.
```

Comment:
**Es wurde ein Fehler behoben, bei dem Artikel-URLs ohne definierten URL-Pfad nicht korrekt generiert werden konnten**
(Der Fehler trat vermutlich nicht nur im Warenkorb auf, sondern überall, wo der URL-Filter angewendet wurde)

Reply:
Due to an error, item URLs haven't been generated correctly without a defined URL path. This has been fixed.

Reply:
thx